manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -1,9 +1,4 @@
 apiVersion: v1
-kind: Namespace
```

Comment:
@dinogun - Do we need a similar change in kruize-crc-openshift.yaml?

Reply:
No, in openshift we own the `openshift-tuning` ns, so we can create and delete it when we start and terminate kruize respectively. Whereas in minikube `monitoring` already exists and cannot be deleted